catroid/src/main/java/org/catrobat/catroid/ui/fragment/BrickCategoryFragment.java

```diff
@@ -71,6 +75,61 @@ public void setBrickAdapter(BrickAdapter brickAdapter) {
 	public void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setHasOptionsMenu(true);
+		initBrickCategories();
+	}
+
+	private void initBrickCategories() {
```

Comment:
is this used somewhere else? otherwise it's unnecessary to put in a separate method